packages/holocron/src/ducks/load.js

```diff
@@ -14,14 +14,10 @@
 
 import { Map as iMap, Set as iSet } from 'immutable';
 import { getModule, getModuleMap } from '../moduleRegistry';
-import { HOLOCRON_STORE_KEY, MODULES_STORE_KEY, REDUCER_KEY } from '../constants';
-
-export const REGISTER_MODULE_REDUCER = '@@holocron/REGISTER_MODULE_REDUCER';
-export const MODULE_LOADED = '@@holocron/MODULE_LOADED';
-export const MODULE_LOAD_FAILED = '@@holocron/MODULE_LOAD_FAILED';
-export const MODULE_LOADING = '@@holocron/MODULE_LOADING';
-export const MODULE_REDUCER_ADDED = '@@holocron/MODULE_REDUCER_ADDED';
```

Comment:
taking out the action types breaks the ducks pattern

Reply:
reverted this changes in 56a4136